ref/feature/mpJwt/examples.adoc

```diff
@@ -13,23 +15,40 @@ The `<mpJwt/>` element is required to configure the processing of the caller’s
         <feature>cdi-2.0</feature>
         <feature>appSecurity-3.0</feature>
     </featureManager>
+----
 
+This is the keystore that is used by SSL and by JWT.
```

Comment:
it might be helpful to combine the code into a single block, with an explanation beforehand, as this is how most of the other config snippets are formatted- unless this one needs to be different. You might also want a subheading for the unified example, like "Basic configuration to implement MicroProfile JWT"